### PR TITLE
feat: switch to use 'open' since 'opn' is deprecated

### PR DESCRIPTION
**Pull request by dkontorovskyy**
- [X] Ready for review
- [X] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [X] Reviewed by Snyk internal team

#### What does this PR do?
`opn` is deprecated package. Maintainers moved code to `open` package.

### CODE REVIEW

**Comment by dkontorovskyy** [comment]
@anthogez yeap :) I've done this already in separate PR :) 
https://github.com/snyk/snyk/pull/1086

File: src/cli/commands/auth/index.ts [hunk 1]
```diff
@@ -64,7 +64,7 @@ async function webAuth(via: AuthCliCommands) {
   try {
     spinner.start();
     await setTimeout(() => {
-      open(urlStr, { wait: false });
```
Comment (dkontorovskyy) [inline src/cli/commands/auth/index.ts:67]: `{wait: false}` is now default behavior for `open` lib

Comment (anthogez) [reply src/cli/commands/auth/index.ts:67]: https://www.npmjs.com/package/open
wait
Type: boolean
Default: false

Comment (dkontorovskyy) [reply src/cli/commands/auth/index.ts:67]: Yes, before default was `true` in `opn` lib.

Comment (dkontorovskyy) [reply src/cli/commands/auth/index.ts:67]: https://github.com/sindresorhus/open/blob/v5.5.0/index.js#L11

**Pull request merged**